docs/output.md

```diff
@@ -26,8 +26,10 @@ This directory contains the zipped results. When unzipping them, the following s
   - `contrasts.tsv`: contrasts used for DE analysis (if provided).
   - `linear_model.txt`: linear model used for DE analysis.
   - `gene_list.txt`: provided list of interesting genes (if provided).
+  - `star_[rsem/salmon]_files.txt`: list of counts files that were provided as input (only if `--input_type` is `rsem` or `salmon`).
+  - `smrnaseq_files.txt`: list of counts files that were provided as input (only if `--input_type` is `smrnaseq`).
 - `gene_counts_tables/`
-  - `raw_gene_counts.txt`: raw gene counts table from the nf-core/rnaseq pipeline and used for the differential gene expression analysis.
+  - `raw_gene_counts.txt`: raw gene counts table from the nf-core/rnaseq pipeline and used for the differential gene expression analysis (only if `--input_type` is `featurecounts` or `smrnaseq`; in the second case, will internally build a counts table from the input files, taking the third column of each .sorted.idxstats file (second column if counting rownames) and combining all values from both hairpin and non-hairpin mature files into one table).
```

Comment:
Actually, "it will be built a counts table" is not really correct. In my expression, I meant something like "the pipeline will build a counts table", just shorter

Reply:
Well, it just sounded odd, like some noun missing, but `the pipeline will internally build` is good.